commit def2f7aad0e7c522be32d98ba08e928259faca9c (refs/changes/80/280/1)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2016-08-12T09:09:49-07:00 (3 years, 2 months ago)
    
    TOOLS-1519 node-sdc-client's make ufds_test target cannot run UFDS tests suite

diff --git a/test/ufds.tests.js b/test/ufds.test.js
similarity index 100%
rename from test/ufds.tests.js
rename to test/ufds.test.js
